student.rb

```diff
@@ -0,0 +1,19 @@
+require './person'
+
+class Student < Person
+  attr_reader :classroom
+
+  def initialize(age:, name: 'Unknown', parent_permission: true)
+    super(name: name, age: age, parent_permission: parent_permission)
+    @classroom = classroom
```

Comment:
- [ ] The highlighted piece of code has some errors as explained in the previous comment, you cannot pass a variable name as a default input parameter without declaring it first, your code will throw up errrors. see the code snippet below on how this should actually look like :+1: :sparkles: 
- code snippet
```ruby
 def initialize(age, classroom, name = "Unknown", parent_permission = true)  
    super(age, name, parent_permission)
    @classroom = classroom
  end
```